mpscenes/obstacles/box_obstacle.py

```diff
@@ -56,6 +56,13 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self.check_completeness()
 
+    def size(self):
+        return [
+            self.length(),
+            self.width(),
+            self.height(),
+        ]
+
```

Comment:
Should we already convert that to a numpy array here? Or is this taken care of in the FullSensor later on?

Reply:
It is taken care of in the fullsensor [here](https://github.com/maxspahn/gym_envs_urdf/blob/1c4506df97497f757c6fad219c1685c56162f872/urdfenvs/sensors/full_sensor.py#L74)